recipe/meta.yaml

```diff
@@ -10,9 +10,8 @@ source:
   sha256: 94f643f1edfaccb57d64b503c7c96f00dec64e8635c054bbaa33855d72c5822d
 
 build:
-  number: 0
+  number: 1
   features:
-    - vc9  # [win and py27]
     - vc14  # [win and py>=35]
   skip: true  # [win and py36]
```

Comment:
Would update this to be `win and (py27 or py36)`. Then [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) the feedstock. This will update the CI configurations accordingly. Need to have `conda` 4.3.x, `conda-build` 2.1.x, `conda-build-all` 1.1.1, and `conda-smithy` 2.4.2 to do the re-rendering.